fenix/app/onboarding.fml.yaml

```diff
@@ -23,6 +27,16 @@ features:
               primary-button-label: juno_onboarding_default_browser_positive_button
               secondary-button-label: juno_onboarding_default_browser_negative_button
 
+            add-search-widget:
+              card-type: add-search-widget
+              title: juno_onboarding_add_search_widget_title
+              body: juno_onboarding_add_search_widget_description
+              link-text: juno_onboarding_add_search_widget_description_link_text
```

Comment:
do we need the privacy link here too?

Reply:
@dnarcese since the privacy link is already in the `Set as default browser` card, would it be also required here?